tests/models/test_dssm.py

```diff
@@ -12,25 +12,46 @@
 #  See the License for the specific language governing permissions and
 #  limitations under the License.
 
+import sys
 import typing as tp
 
 import numpy as np
 import pandas as pd
 import pytest
-from lightning_fabric import seed_everything
+
+try:
+    from lightning_fabric import seed_everything
+except ImportError:
+    pass
+
+try:
+    import pytorch_lightning  # noqa  # pylint: disable=unused-import
+
+    filter_decorator = pytest.mark.filterwarnings("ignore::pytorch_lightning.utilities.warnings.PossibleUserWarning")
```

Comment:
Let's rename it to `filter_warnings_decorator`

Reply:
Fixed [3b5e594](https://github.com/MobileTeleSystems/RecTools/pull/227/commits/3b5e594e528afca7d6e2af5b6154fb1a9b571fd3)